exercises/scale-generator/description.md

```diff
@@ -1,26 +1,24 @@
 # Description
 
-Given a tonic, or starting note, and a set of intervals, generate
-the musical scale starting with the tonic and following the
-specified interval pattern.
```

Comment:
Seems to me we should give the lay of the land before giving the problem statement.
Additionally there are two tasks for this exercise but IMO that was not clear from this statement.

I moved the task directives down below the relevant background info and spelled out the requirements of each task.